index.d.ts

```diff
@@ -1,5 +1,5 @@
-declare module 'plaid' {
-  import { CoreOptions } from 'request';
+declare module "plaid" {
```

Comment:
why change these to `"`? Our internal linter prohibits them (probably just for the sake of consistency)... is there any concrete benefit to using the double quote? If not, lets leave it out of this change...

Reply:
I didn't realize prettier had run.